File: broadcastreceivers/src/main/java/com/smartherd/broadcastreceivers/MainActivity.java
package com.smartherd.broadcastreceivers;


import android.content.IntentFilter;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.smartherd.broadcastreceivers.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {


    OrderedReceiver1 orderedReceiver1 = new OrderedReceiver1();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        IntentFilter filter = new IntentFilter("com.codinginflow.EXAMPLE_ACTION");
        registerReceiver(orderedReceiver1, filter);
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        unregisterReceiver(orderedReceiver1);
    }
}

File: viewpagerwithnavigation/src/main/java/com/smartherd/viewpagerwithnavigation/MainActivity.kt
package com.smartherd.viewpagerwithnavigation

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.smartherd.viewpagerwithnavigation.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        supportActionBar?.hide()
    }
}